frontend/src/container/NewDashboard/ComponentsSlider/index.tsx

```diff
@@ -56,15 +59,29 @@ function DashboardGraphSlider(): JSX.Element {
 		}
 	};
 
+	const handleCardClick = (panelType: PANEL_TYPES): void => {
+		onClickHandler(panelType)();
```

Comment:
can be simplied to `onClickHandler(panelType);`

with above function definition as - `const onClickHandler = (name: PANEL_TYPES) => {`